app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] Kindly ensure that there is a corresponding form in the _views/comments/new.html.erb_ file which accepts input for creating _comments_.

Reply:
done, thanks!